docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,629 @@
+
+Type `q` to quit viewing the list of branches.
+
+### 6. Checkout your feature branch
+
+You want to start making changes in your feature branch and not your `master` branch, so switch to your `feature` branch.
+
+Type
+
+```
+git checkout emonddr-doc-changes
+```
+
+
+![submit_pr_clone_8.png](./imgs/submit_pr_clone_8.png)
+
+You have successfully switched to your feature branch, and you can also confirm this by typing
+
+```
+git branch -a
+```
+
+![submit_pr_clone_9.png](./imgs/submit_pr_clone_9.png)
+
+Type `q` to quit viewing the list of branches.
+
+
+### 7. Set up remote tracking
+
+The local copy of your repository needs to track both your remote repository and the original repository from which it was forked. By default, when cloning, a link is created back to your copy but not the original repository.
+
+Let's find out what remotes currently exist.
+
+Type
+
+```
+git remote
+```
+
+![submit_pr_remote_tracking_1.png](./imgs/submit_pr_remote_tracking_1.png)
+
+There should be one remote named **origin** and it points to the forked remote repository.
+
+If a remote named **upstream** (pointing to the original remote repository) does not exist, you need to create it.
+
+In your browser, navigate to original repository  <https://github.com/strongloop/loopback-next> and ensure that the `master` branch is selected.
+
+
+![submit_pr_remote_tracking_2.png](./imgs/submit_pr_remote_tracking_2.png)
+
+Click on the `Clone or download` button, leave the cloning option as `Clone with SSH` and press the `copy to clipboard` icon.
+
+![submit_pr_remote_tracking_3.png](./imgs/submit_pr_remote_tracking_3.png)
+
+
+Return to your terminal window and type :
+
+```
+git remote add upstream git@github.com:strongloop/loopback-next.git
+```
+
+then type :
+
+```
+git remote
+```
+
+there should now be two remotes : **origin** and **upstream**.
+
+![submit_pr_remote_tracking_4.png](./imgs/submit_pr_remote_tracking_4.png)
+
+Typing
+
+```
+git remote -v
+```
+
+provides more details.
+
+![submit_pr_remote_tracking_5.png](./imgs/submit_pr_remote_tracking_5.png)
+
+### 8. Install dependencies
+
+The `loopback-next` has several package dependencies.
+
+Type `npm install` to install them.
+
+![submit_pr_npm_install_1.png](./imgs/submit_pr_npm_install_1.png)
+
+
+### 9. Do some work in your local feature branch directory
+
+Whether you are contributing to code or documentation, be sure to make all your changes inside in the local feature branch directory.
+
+Be sure to read, understand, and abide by the instructions provided in [Contributing code](https://loopback.io/doc/en/contrib/code-contrib.html), [Contributing to docs](https://loopback.io/doc/en/contrib/doc-contrib.html), and
+[Contributing to LoopBack](https://github.com/strongloop/loopback-next/blob/master/docs/CONTRIBUTING.md)
+
+### 10. Verifying your code changes
+
+Before `staging your changes` (the next step), be sure your files are free of formatting, syntax, and logical/execution errors.
```

Comment:
`Staging your changes` is the step after the next one

Reply:
@nabdelgadir, perhaps 
"
Before the `Staging your changes` step, be sure your files are free of formatting, syntax, and logical/execution errors.
"
?